src/main/scala/exercises/00_HelloWorld.scala

```diff
@@ -64,5 +64,6 @@ object HelloWorld {
    * This exercise is taken from ScalaLabs:
    * 		https://github.com/scala-labs/scala-labs/blob/master/labs/src/main/scala/org/scalalabs/basic/lab01/HelloWorldExercise.scala
    */
-  val message: String = "Change me"
-}
\ No newline at end of file
+  val message: String = "Hello world!"
+  println(HelloWorld.message);
```

Comment:
Print statements are good for debugging, but I don't think this is necessary for this exercise